django_celery_results/urls.py

```diff
@@ -6,21 +6,49 @@
     URL  to :func:`~celery.views.task_status`.
 """
 
-from django.conf.urls import url
+from django.conf.urls import path, register_converter
```

Comment:
i was thinking about importing from django.urls as we support 2.2+ in master

Reply:
good with me!